src/run-openwrt.sh

```diff
@@ -31,7 +31,9 @@ start_sqm_section() {
     export IFACE=$(config_get "$section" interface)
 
     [ -z "$RUN_IFACE" -o "$RUN_IFACE" = "$IFACE" ] || return
-    [ "$(config_get "$section" enabled)" -eq 1 ] || return
+    if [ ! "$(config_get "$section" enabled)" -eq 1 ]; then
+        return
+    fi
```

Comment:
Erm, the before and after here are logically equivalent, but happens to change the return code of the test invocation, which then becomes the return code of the function. The right way to fix this is just to change the `return` into `return 0`.